slides.Rmd

```diff
@@ -198,59 +264,50 @@ smr01 <- dbGetQuery(smra_connection,
 ```
 Instructions on using R to connect to SMRA can be found on the [ISD Website](https://www.isdscotland.org/About-ISD/Methodologies/_docs/Using-R-with-SMRA-V1-1.pdf)
 
-## exporting data
-
-very similar to reading in data e.g.
-
-- `readr::write_csv(object, "file_name.csv")`
-- `haven::write_sav(object, "file_name.sav")`
 
-## Data Exploration - Data Frame 
+## END OF BLOCK 1
 
-Let's do some data exploration.
 
-As a first step, it is important to know the size of the dataframe. 
+## START BLOCK 2
 
 
-```{r,  eval=FALSE,echo=TRUE}
-dim(df)
-```
-We can check the number of rows and columns for the `borders` dataframe 
-
+## Data Exploration
```

Comment:
When Graeme and I delivered this we skipped over the base R stuff quickly and then moved straight to dplyr / tidyverse.

Using dplyr::count() instead of table, dplyr::summarise(mean()) etc. and then janitor::tabyl() for crosstabs. This does mess up the flow a bit with dplyr being the next block but you could argue that these commands are for data exploration i.e. just printing stats to the screen and the stuff in the next block is actual manipulation.

Reply:
I like this idea but it doesn't seem to work on the iris dataset. Happy to change the dataset if you have an example it works well on! I just picked the iris one at random but am on leave after today until Tuesday so might not have time to fix this myself.